src/util.rs

```diff
@@ -56,3 +62,15 @@ where
     }
     i
 }
+
+/// Fill a file with 0.
+pub fn fill_file(file: &mut impl std::io::Write, mut size: u64) -> Result<()> {
```

Comment:
Why not just `file.set_len(size)`?

Reply:
I'll try

Reply:
This function has been removed for now. We will directly call `set_len` on `File`, instead of creating this new function to handle that.